### PR TITLE
DISTPG-385 Release notes 14.2

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   docs/release-notes.md
	modified:   mkdocs-base.yml
docs/release-notes-v14.2.md

### CODE REVIEW

File: mkdocs-base.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaDistributionPostgreSQL-14.pdf'
           cover_title: 'Percona Distribution for PostgreSQL Documentation'
-          cover_subtitle: 14.1 (December 7, 2021)
+          cover_subtitle: 14.2 (March 21, 2022)
```
Comment (EvgeniyPatlan) [inline mkdocs-base.yml]: date is in future

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: mkdocs-base.yml (hunk 1 above)
Comment (patrickbirch) [reply mkdocs-base.yml]: Fixed.

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by patrickbirch — APPROVED** [review]
Good

**Review by LenzGr — APPROVED** [review]
LGTM, thank you!

**Pull request merged**